app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -164,19 +164,29 @@ private void init() {
 
         if (callback.getIndexInViewFlipper(this) == 0) {
             btnPrevious.setEnabled(false);
+
             btnPrevious.setAlpha(0.5f);
         } else {
             btnPrevious.setEnabled(true);
             btnPrevious.setAlpha(1.0f);
         }
 
         //If this is the last media, we have nothing to copy, lets not show the button
-        if (callback.getIndexInViewFlipper(this) == callback.getTotalNumberOfSteps()-4) {
-            btnCopyToSubsequentMedia.setVisibility(View.GONE);
-        } else {
+
+        if (callback.getIndexInViewFlipper(this) != callback.getTotalNumberOfSteps()-4){
```

Comment:
Would you mind keeping the `==` and flipping the GONE/VISIBLE calls? Double-negations are confusing. Thanks!